android/app/src/main/java/com/ohmymeme/app/CloudSync.kt

```
@@ -0,0 +1,1548 @@
+                    local.inputStream().use { ins ->
+                        val buf = ByteArray(65536)
+                        while (true) {
+                            val n = ins.read(buf)
+                            if (n <= 0) break
+                            it.getOutputStream().write(buf, 0, n)
+                        }
+                    }
+                }
+                cmd("STOR $remotePath")
+                true
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)